Extensions/DialogueTree/dialoguetools.js

```diff
@@ -482,11 +482,14 @@ gdjs.dialogueTree.goToNextDialogueLine = function() {
     this.dialogueData = this.dialogue.next().value;
   } else if (gdjs.dialogueTree._isLineTypeOptions()) {
     this.dialogueDataType = 'options';
+    this.dialogueText = ' ';
```

Comment:
@blurymind 
Alright, just double checking that you want this line too? Will merge then when I get your go/change.

Reply:
yes I do, but actually doesnt need that extra space in it. I will update it sorry.

Will do a pr to bondagejs soon. I think I know where this happens

Reply:
updated now. Thank you for reviewing this :)

Reply:
👍